src/Serilog.Extensions.Logging/Extensions/Logging/EventIdPropertyCache.cs

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
Any particular reason for the .NET Foundation header?

Reply:
No particular reason. I just copied this header from the [SerilogLogger ](
https://github.com/serilog/serilog-extensions-logging/blob/dev/src/Serilog.Extensions.Logging/Extensions/Logging/SerilogLogger.cs) class. 

I noticed the files in repository have different copyright headers and wondered if they should be consolidated someday.